policies-admin/index.qmd

```diff
@@ -2,84 +2,4 @@
 title: "Policies & Administration"
```

Comment:
```suggestion
title: "Access & Administration"
```

Is it also possible to rename the file to `access-admin/index.qmd`, and have an alias so the url doesn't break? Thanks

Reply:
I can certainly rename it, and will look at redirects. Thanks!

Reply:
I've added aliases to all pages in `access-admin`